modules/nf-core/gget/gget/main.nf

```diff
@@ -0,0 +1,38 @@
+process GGET_GGET {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gget=0.27.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gget:0.27.2--pyhdfd78af_0':
+        'quay.io/biocontainers/gget:0.27.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(files)
+
+    output:
+    tuple val(meta), path("*")     , emit: file, optional: true
```

Comment:
This will emit all files (including the `json` and `csv` files you would like to emit separately)

Reply:
This should be possible with glob patterns I think?

Reply:
I do remember that it has been done in a module before but I can't seem to find it. You could indeed look for `${prefix}.${suffix}` in the first one. Do the json and csv have different names? You could maybe rename those so it's easier to match these separately?